app/workers/hca/service_job.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+module HCA
+  class ServiceJob
+    include Sidekiq::Worker
+
+    sidekiq_retries_exhausted do |msg, _e|
```

Comment:
Should we configure the retries in a special way for this job, or do we think the default retry stuff is good?